spec/src/modules/quizzes.js

```diff
@@ -92,21 +92,42 @@ describe(`ConstructorIO - Quizzes${bundledDescriptionSuffix}`, () => {
       });
     });
 
-    it('Should return a result provided a valid apiKey, quizId and versionId', () => {
-      const versionId = 'e03210db-0cc6-459c-8f17-bf014c4f554d';
+    it('Should return a result provided a valid apiKey, quizId and quizVersionId', () => {
+      const quizVersionId = 'e03210db-0cc6-459c-8f17-bf014c4f554d';
       const { quizzes } = new ConstructorIO({
         apiKey: quizApiKey,
         fetch: fetchSpy,
       });
 
-      return quizzes.getQuizNextQuestion(validQuizId, { versionId }).then((res) => {
+      return quizzes.getQuizNextQuestion(validQuizId, { quizVersionId }).then((res) => {
         const requestedUrlParams = helpers.extractUrlParamsFromFetch(fetchSpy);
 
-        expect(res).to.have.property('version_id').to.be.an('string').to.equal(versionId);
+        expect(res).to.have.property('version_id').to.be.an('string').to.equal(quizVersionId);
         expect(res).to.have.property('next_question').to.be.an('object');
         expect(res.next_question.id).to.equal(1);
         expect(res.next_question.options[0].id).to.equal(1);
-        expect(requestedUrlParams).to.have.property('version_id').to.equal(versionId);
+        expect(requestedUrlParams).to.have.property('quiz_version_id').to.equal(quizVersionId);
+      });
+    });
+
+    it('Should return a result provided a valid apiKey, quizId and quizVersionId, quizSessionId', () => {
+      const quizVersionId = 'e03210db-0cc6-459c-8f17-bf014c4f554d';
+      const quizSessionId = '123;';
+      const { quizzes } = new ConstructorIO({
+        apiKey: quizApiKey,
+        fetch: fetchSpy,
+      });
+
+      return quizzes.getQuizNextQuestion(validQuizId, { quizVersionId, quizSessionId }).then((res) => {
+        const requestedUrlParams = helpers.extractUrlParamsFromFetch(fetchSpy);
+
+        expect(res).to.have.property('version_id').to.be.an('string').to.equal(quizVersionId);
```

Comment:
Same as above, I think we'll want to change the expectation to check `quiz_version_id`

I think the `quiz_session_id` is returned in the response as well, could we add an expectation around that?